mxcube3/routes/scutils.py

```diff
@@ -109,17 +113,17 @@ def mount_sample(beamline_setup_hwobj,
                     msg = "Manual centring used, waiting for" +\
                           " user to center sample"
                     log.warning(msg)
-                    dm.start_centring_method(dm.CENTRING_METHOD_MANUAL)
+                    dm.startCentringMethod(dm.MANUAL3CLICK_MODE)
```

Comment:
this does not exist in the `GenericDiffractometer` (snakecase yes )

Reply:
Yes I noticed, its unfortunately what we have on our beamlines so there is not much we can do about it for the moment. Both the Minidfiff and Microdiff classes uses the camel case version, which one are you using ?

Reply:
we are using the `snake_case`. I am fine with a `hasattr`, like we have done before in similar cases.

Reply:
This could get quite problematic, since for instance camel case is used in the queue_entry and perhaps in other places. The interface of GenericDiffractomter is not really applied, which is 
a problem in itself. This is I believe is due to that the work with it is not complete yet.

I noticed that you already have some camel case aliases in your diffractometer so Id suggest that you add one for startCentringMethod aswell so that its backward compatible with the rest of the queue.

We will continue the discussion with the new interface for hardware objects 3.0

Marcus

Reply:
I guess I can do that